internal/worker/apiserver/manifold.go

```diff
@@ -301,18 +301,10 @@ func (config ManifoldConfig) start(ctx context.Context, getter dependency.Getter
 		return cmd.Main(jujuCmd, ctx, strings.Split(cmdPlusARgs, " "))
 	}
 
-	// Get the state pool after grabbing dependencies so we don't need
-	// to remember to call Done on it if they're not running yet.
-	statePool, _, err := stTracker.Use()
```

Comment:
todo: remove the `stTracker` variable. Only used here.

Reply:
Will be done in another ticket.